### PR TITLE
(Chore) Exception message assignment

### PR DESCRIPTION
**Pull request by janjaap**
This PR fixes an issue where the application would throw error unrelated to pending actions. This was due to the fact that in the `useFetch` custom hook, the error message was set on a freezed object.

### CODE REVIEW

**Comment by janjaap** [comment]
What do you mean by immutable? You example doesn't show anything immutable.

**Comment by aditudorache** [comment]
Bad choice of words, I meant create a new instance of the object and pass it through.

**Comment by janjaap** [comment]
Ah, like that. Well, that would make testing significantly harder as I cannot assets that a thrown error is the actual error that is available after hook update.

**Comment by janjaap** [comment]
I've added a couple of assertions that verify that the message has been set on the thrown error.

File: src/hooks/__tests__/useFetch.test.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { getErrorMessage } from 'shared/services/api/api';
 
 import useFetch from '../useFetch';
 
-jest.mock('shared/services/api/api');
+// jest.mock('shared/services/api/api');
```
Comment (aditudorache) [inline src/hooks/__tests__/useFetch.test.js]: forgot to remove?

Comment (janjaap) [reply src/hooks/__tests__/useFetch.test.js]: No, it's not there any more.

**Pull request merged**